build/nuget-org.packages.config

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.CodeAnalysis" />
+  <package id="Microsoft.CodeAnalysis.Common" />
+  <package id="Microsoft.CodeAnalysis.Compilers" />
+  <package id="Microsoft.CodeAnalysis.CSharp" />
+  <package id="Microsoft.CodeAnalysis.CSharp.Features" />
+  <package id="Microsoft.CodeAnalysis.CSharp.Scripting" />
+  <package id="Microsoft.CodeAnalysis.CSharp.Workspaces" />
+  <package id="Microsoft.CodeAnalysis.EditorFeatures" />
+  <package id="Microsoft.CodeAnalysis.EditorFeatures.Text" />
+  <package id="Microsoft.CodeAnalysis.Features" />
+  <package id="Microsoft.CodeAnalysis.Scripting" />
+  <package id="Microsoft.CodeAnalysis.Scripting.Common" />
+  <package id="Microsoft.CodeAnalysis.VisualBasic" />
+  <package id="Microsoft.CodeAnalysis.VisualBasic.Features" />
+  <package id="Microsoft.CodeAnalysis.VisualBasic.Workspaces" />
+  <package id="Microsoft.CodeAnalysis.Workspaces.Common" />
+  <package id="Microsoft.DiaSymReader.PortablePdb" />
```

Comment:
Does DiaSymReader.PortablePdb always ship on same cadence as Roslyn? Or is this handled separately? (I thought I heard that this was moving to a separate repo?)